test/functional/features/shared-resource-behaviour.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Verify the shared resource CSI driver works as expected
+
+    As an application developer using OpenShift
+    I want to verify the projected resource CSI driver shares work on OpenShift
```

Comment:
I think that this should be more specific. "works on OpenShift" or "works as expected" is probably to wide.
Maybe consider using more descriptive sentence of what we expect to work:
- can be installed on a running cluster
by cheching that:
- driver's daemonset has the expected replicas count
- driver pods starts and are ready
- driver pods logs contains not error etc...